src/utils/tokens/instructions.js

```diff
@@ -73,7 +74,7 @@ export function initializeMint({
         decimals,
         mintAuthority: mintAuthority.toBuffer(),
         freezeAuthorityOption: !!freezeAuthority,
-        freezeAuthority: (freezeAuthority || new PublicKey()).toBuffer(),
+        freezeAuthority: (freezeAuthority || Keypair.generate().publicKey).toBuffer(),
```

Comment:
This change is not equivalent. It should be be `PublicKey.default`.

Reply:
Updated thanks. Just for my understanding, the `PublicKey.default` means we're basically giving up freezeAuthority here?

Reply:
Correct.